docs/integrate-cardano/testnet-faucet.md

```diff
@@ -8,19 +8,27 @@ image: ../img/og/og-developer-portal.png
 
 The testnet faucet is a service that provides test ada (tAda) to users of the Cardano testnet. These tokens have no value, but they enable users to experiment with Cardano testnet features without spending real ada on the mainnet. You can use test ada to [mint native tokens](../native-tokens/minting), to [play around with Cardano wallets](creating-wallet-faucet) or to [learn how to operate a stake pool](../operate-a-stake-pool/).
 
-## How to get test ada
-1. Default funds are in `tAda`. If you'd like to test the native token functionality, select `Testcoin` from the dropdown menu.
-1. Enter the address of the account where you want to top up funds.
-1. If you have been issued with an API key, please enter this to access any additional funds you may have been allocated.
-1. Confirm the `I'm not a robot` box and solve the captcha if needed.
-1. Click on `Request` and the funds will be in the testnet account you specified within a few minutes. Use the [Cardano testnet explorer](https://explorer.cardano-testnet.iohkdev.io/) in case you want to check any transactions on the Cardano testnet.
-
-<div id="faucetcontainer">
-<iframe name="iframe" height="300" width="100%" scrolling="no" src="https://testnets.cardano.org/en/testnets/cardano/tools/faucet/" class="faucet"></iframe>
-</div>
-
+## Testnets faucet
 
+This portal allows you to send test ada to your wallet. You can access it [here](https://docs.cardano.org/cardano-testnet/tools/faucet).
 
 :::note Return the test ada
 Send your test tokens to this address: `addr_test1qqr585tvlc7ylnqvz8pyqwauzrdu0mxag3m7q56grgmgu7sxu2hyfhlkwuxupa9d5085eunq2qywy7hvmvej456flknswgndm3`
-:::
\ No newline at end of file
+:::
+
+## How to get test ada 
+1. Define your environment. You can get tada for either the `preview` or `prepod testnet`.
+2. Define your action. Select `Receive test ADA` in this case. You can also ask for `pool delegation` if you are managing a pool (learn [more](https://developers.cardano.org/docs/operate-a-stake-pool/)).
+3. Enter the address of the account where you want to top up funds.
+4. If you have been issued with an API key, please enter this to access any additional funds you may have been allocated.
+5. Confirm the `I'm not a robot` box and solve the captcha if needed.
+6. Click on `Request` and the funds will be in the testnet account you specified within a few minutes. Use the [Cardano testnet explorer](https://explorer.cardano-testnet.iohkdev.io/) in case you want to check any transactions on the Cardano testnet.
+
+<!-- Commented iframe, but please feel free to test the w x h to fit it to the browser -->
+<!-- <div id="faucetcontainer">
+<iframe name="iframe" height="500" width="150%" src="https://testnets.cardano.org/en/testnets/cardano/tools/faucet/" class="faucet"></iframe>
```

Comment:
Has anyone tested to see if simply setting `width="100%"` instead will fit the `<iframe>` in the body column, to see if it fixes https://github.com/cardano-foundation/developer-portal/issues/812?  (I understand there may not currently be a working faucet for Preview & Pre-Production)

Reply:
p.s. feel free to disregard as I'm a latecomer to this.  Looks like width had already been set to 100% and I can't see why it was changed to 150%.  I must have missed something since I don't fully understand why the faucet is being commented out rather than updating the embed:

- I do see there is a working faucet which seems adjusted to the new Testnet environments: https://docs.cardano.org/cardano-testnet/tools/faucet
- which imports this code: https://github.com/input-output-hk/cardano-documentation/blob/master/src/components/Faucet.js
- from this page: https://github.com/input-output-hk/cardano-documentation/blob/master/content/05-cardano-testnet/06-tools/01-faucet.mdx

... but I'd have no idea yet how to include it in the environment here on the Dev Portal.  If there's any chance in fixing this with this PR it seems like it would be worth it (again, apologies if I've missed something).